tensorboard/plugins/histogram/summary_v2.py

```diff
@@ -43,32 +43,41 @@ def histogram(name, data, step=None, buckets=None, description=None):
 
     See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
 
-    Logs a histogram to the current log directory for later analysis in
-    TensorBoard's 'Histograms' and 'Distributions' dashboards (one histogram
-    will appear in both). Like `tf.summary.scalar` points, each histogram is
-    associated with a `step` and a `name`. All the histograms with the same
-    `name` constitute a time series of histograms.
+    Writes a histogram to the current default summary writer, for later analysis
+    in TensorBoard's 'Histograms' and 'Distributions' dashboards (data written
+    using this API will appear in both places). Like `tf.summary.scalar` points,
+    each histogram is associated with a `step` and a `name`. All the histograms
+    with the same `name` constitute a time series of histograms.
+
+    When higher order `Tensor`s are flattened to produce a histogram of all
```

Comment:
This line seems like it's missing a word?  Maybe you meant something like "When _called with_ higher order tensors, they are flattened..."?

If that's the case, it's a bit imprecise what "higher order" actually means here.  If we do mention flattening I might do so in the Args section below, e.g. extend the `data` comment to

> A `Tensor` of any shape. The histogram is computed over its elements, which must be castable to `float64`.

Or if we want a general comment up here, maybe phrase it more like "The histogram is calculated over the elements of the given tensor without regard to its shape or rank."

Reply:
I like the wording, done.